.github/workflows/ci.yml

```diff
@@ -64,9 +64,6 @@ jobs:
     name: Test mOTUs with workflow parameters
     if: ${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'nf-core/taxprofiler') }}
     runs-on: ubuntu-latest
-    env:
-      NXF_VER: ${{ matrix.nxf_ver }}
-      NXF_ANSI_LOG: false
```

Comment:
Is this a necessary change to fix this issue?

Reply:
Apparetly, it's a duplicate of this (further below), and I think leads to conflicts. at least when I include it it crashes, I remove it it works 🤷. It makes sense anyway to keep it in sync with template (and the CI test above).

```
      matrix:
        NXF_VER:
          - "21.10.3"
          - "latest-everything"
 `